guides/network/using-middleware.md

```diff
@@ -1,52 +0,0 @@
-# Using middleware
```

Comment:
What I'll do is to remove the file from the ToC while keeping it as a page to allow links to it.